pkg/plugins/golang/v4/scaffolds/internal/templates/webhooks/webhook_suitetest.go

```diff
@@ -61,17 +76,25 @@ func (f *WebhookSuite) SetTemplateDefaults() error {
 
 	f.TemplateBody = fmt.Sprintf(webhookTestSuiteTemplate,
 		machinery.NewMarkerFor(f.Path, importMarker),
-		admissionImportAlias,
+		f.Resource.ImportAlias(), admissionImportAlias,
 		machinery.NewMarkerFor(f.Path, addSchemeMarker),
 		machinery.NewMarkerFor(f.Path, addWebhookManagerMarker),
 		"%s",
 		"%d",
 	)
 
-	// If is multigroup the path needs to be ../../.. since it has the group dir.
-	f.BaseDirectoryRelativePath = `"..", ".."`
-	if f.MultiGroup && f.Resource.Group != "" {
-		f.BaseDirectoryRelativePath = `"..", "..",".."`
+	if f.IsLegacyPath {
+		// If is multigroup the path needs to be ../../.. since it has the group dir.
+		f.BaseDirectoryRelativePath = `"..", ".."`
+		if f.MultiGroup && f.Resource.Group != "" {
+			f.BaseDirectoryRelativePath = `"..", "..", ".."`
+		}
+	} else {
+		// If is multigroup the path needs to be ../../.. since it has the group dir.
```

Comment:
The comment says ../../.. but its actually ../../../.. here.

Reply:
We just did not update the comment after the changes
We can address this nit.